iowrappers/jobs.go

```diff
@@ -0,0 +1,83 @@
+package iowrappers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+type JobStatus string
+
+const (
+	JobStatusNew       JobStatus = "new"
+	JobStatusCreated   JobStatus = "created"
+	JobStatusRunning   JobStatus = "running"
+	JobStatusFailed    JobStatus = "failed"
+	JobStatusCompleted JobStatus = "completed"
+
+	JobExpirationTime = time.Hour
```

Comment:
so this is `1` hour ? any reason, or this can also be fine tuned ?

Reply:
changed to 24 hours, so that admins can inspect.